test/metabase/api/user_test.clj

```diff
@@ -326,9 +329,20 @@
 ;;; |                                      Updating a User -- PUT /api/user/:id                                      |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(defn include-personal-collection-name
```

Comment:
Is this function being used anywhere?

Reply:
Yeah, it's tagged as the hydrate function for `:personal_collection_name` which is used below to verify the name of the personal collection would be changed after updating the user.

```clojure
(defn include-personal-collection-name
  {:hydrate :personal_collection_name}
  ...
```